readme.md

```diff
@@ -1388,6 +1389,21 @@ export default () => <div>
 </div>
 ```
 
+### Starting a dev server on alternative IP
+
+To start the dev server using a hostname different from the defualt `localhost` you can use the provided `--hostname or -H` option with `next dev`.
```

Comment:
`defualt` => `default`

Reply:
To start the development server using a different default hostname you can use  `--hostname or -H` option with `next dev`.